### ISSUE TITLE
Updates Analysis and Network visualisation features 

### ISSUE THREAD

**Issue by GCamp6** [nealhaddaway/citationchaser#40]
On the app version of the package at https://estech.shinyapps.io/citationchaser/ the Analysis and Network visualisation features are being inactive since a while (_We're updating the citation analysis features - we'll be back soon!_). Do you have feedbacks on the implementation process?

**Comment by nealhaddaway** [comment]
Yes - we're hoping to reinstate those during a major upgrade planned during ESH2024 at the end of this month. More details to follow! Thanks
________________________________
From: GCamp6 ***@***.***>
Sent: 09 December 2024 12:01
To: nealhaddaway/citationchaser ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nealhaddaway/citationchaser] Updates Analysis and Network visualisation features (Issue #40)


On the app version of the package at https://estech.shinyapps.io/citationchaser/ the Analysis and Network visualisation features are being inactive since a while (We're updating the citation analysis features - we'll be back soon!). Do you have feedbacks on the implementation process?
